sdk/modelsrepository/Azure.Iot.ModelsRepository/src/Fetchers/RemoteModelFetcher.cs

```diff
@@ -13,6 +13,10 @@
 
 namespace Azure.Iot.ModelsRepository.Fetchers
 {
+    /// <summary>
+    /// The <c>RemoteModelFetcher</c> is an implementation of IModelFetcher
+    /// for supporting http[s] based model content fetching.
```

Comment:
We don't force HTTPS?

Reply:
We use https for the IoT global endpoint, but its not necessary to force https in user defined environments. For example if they want to test against a local webserver.